Kickstarter-iOS/Views/CreatorBylineView.swift

```diff
@@ -0,0 +1,148 @@
+import KsApi
+import Library
+import Prelude
+import UIKit
+
+private enum Layout {
+  enum ImageView {
+    static let minHeight: CGFloat = 36.0
+    static let minWidth: CGFloat = 36.0
+  }
+}
+
+final class CreatorByLineView: UIView {
```

Comment:
Given that _byline_ is just one word, I think it'd be nice if we could rename instances of `CreatorByLine` -> `CreatorByline`. This should be quick if you do a case-sensitive find/replace in the project.